File: 01. JAVA/04. Tipos de datos/TiposDeDatos.java
package tiposdedatos;

public class TiposDeDatos {

    public static void main(String[] args) {
        
        //Entero de 1 byte - 8bits (-128 a 127)
        byte vBite2 = 127;
        
        //Entero de 2 bytes -16 bits (-32768 a 32767)
        short vShort2 = 32767;
        
        //Entero de 4 bytes - 32 bits(-2147483648 a 2147483647)
        int vInt2 = 2147483647;
        
        //Entero de 4 bytes - 64 bits(-9223372036854775808 a 9223372036854775807)
        long vLong2 = 9223372036854775807l;
        
        //Decimal simple de 4 bytes - 64 bits (simple presicion)
        float vFloat1 = (float) 123.456;
        
        //Decimal doble de 8 bytes (Doble presicion)
        double vDouble1 = 123.456;
        
        //Caracter simple de 2 bytes
        char vChar1 = 'a';
        
        //Cadena de caracteres
        String vString1 = "cadena de texto";
        
        //valor verdadero o falso;
        boolean vBoolean1 = true;
        
        System.out.println("Bite: "+vBite2);
        System.out.println("Short: "+vShort2);
        System.out.println("Bite: "+vInt2);
        System.out.println("Long: "+vLong2);
        System.out.println("Float: "+vFloat1);
        System.out.println("Double: "+vDouble1);
        System.out.println("Char: "+vChar1);
        System.out.println("String: "+vString1);
        System.out.println("Boolean: "+vBoolean1);
        
    }
    
}

File: 05. Lenguajes no clasificados/01. C#/04. Tipos de datos/Tipos de datos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TiposDeDatos
{
    class Program
    {
        static void Main(string[] args)
        {

            //Entero de 1 byte (-128 a 127)
            byte vBite2 = 127;
        
            //Entero de 2 bytes (-32768 a 32767)
            short vShort2 = 32767;
        
            //Entero de 4 bytes (-2147483648 a 2147483647)
            int vInt2 = 2147483647;
        
            //Entero de 4 bytes (-9223372036854775808 a 9223372036854775807)
            long vLong2 = 9223372036854775807l;
        
            //Decimal simple de 4 bytes
            float vFloat1 = (float) 123.456;
        
            //Decimal doble de 8 bytes
            double vDouble1 = 123.456;
        
            //Caracter simple de 2 bytes
            char vChar1 = 'a';
        
            //Cadena de caracteres
            String vString1 = "cadena de texto";
        
            //valor verdadero o falso;
            bool vBoolean1 = true;
        
            Console.WriteLine("Bite: "+vBite2);
            Console.WriteLine("Short: "+vShort2);
            Console.WriteLine("Bite: "+vInt2);
            Console.WriteLine("Long: "+vLong2);
            Console.WriteLine("Float: "+vFloat1);
            Console.WriteLine("Double: "+vDouble1);
            Console.WriteLine("Char: "+vChar1);
            Console.WriteLine("String: " + vString1);
            Console.WriteLine("Boolean: " + vBoolean1);

            Console.In.ReadLine();

        }
    }
}
